src/server/pfs/server/trigger.go

```diff
@@ -59,7 +60,7 @@ func (d *driver) triggerCommit(
 // isTriggered checks to see if a branch should be updated from oldHead to
 // newHead based on a trigger.
 func (d *driver) isTriggered(txnCtx *txncontext.TransactionContext, t *pfs.Trigger, oldHead, newHead *pfs.CommitInfo) (bool, error) {
-	if t == nil {
+	if t == nil || t.CronSpec != "" {
```

Comment:
the other trigger condition types can be used non-exclusively right? Just calling out that this diverges from the `All` behavior

Reply:
Yeah, that seems implicit in the description of the new field in the proto, but I can make it more explicit.